flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptivebatch/AdaptiveBatchSchedulerTest.java

```diff
@@ -119,43 +119,62 @@ void testAdaptiveBatchScheduler() throws Exception {
         assertThat(sink.getParallelism()).isEqualTo(10);
 
         // check aggregatedInputDataBytes of each ExecutionVertex calculated.
-        checkAggregatedInputDataBytesIsCalculated(sinkExecutionJobVertex);
+        checkAggregatedInputDataBytesIsCalculated(sinkExecutionJobVertex, 26_000L);
```

Comment:
This magic number is hard to understand, would you add some comments to explain it ?